.gitignore

```diff
@@ -5,3 +5,4 @@
 .opt-out
 coverage
 node_modules
+.vscode
```

Comment:
Is this meant to be checked in?

Reply:
That's all I got, 🚀

Reply:
didn't want to have my vscode config checked in (I think it was to have the script run with debugger

```
{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "node",
      "program": "${workspaceFolder}/src/index.js",
      "args": ["find", "blue.500"]
    }
  ]
}
```